fanos.go

```diff
@@ -64,7 +64,8 @@ func (s *FANOSShell) Run(ctx context.Context, r io.Reader) error {
 
 	var buf bytes.Buffer
 	buf.WriteString("EVAL ")
-	_, err := io.Copy(&buf, r); if err != nil {
+	_, err := io.Copy(&buf, r)
+	if err != nil {
```

Comment:
I'm curious, did gofmt break this up automatically?

Reply:
yep, I never touched that line!